### DISCUSSION TITLE
Customizing ports

### DISCUSSION THREAD

**Discussion by shizenyasuya** [mrlt8/docker-wyze-bridge#812] [Q&A] [answered]
I'm having trouble viewing the web UI, and I think since I refreshed my image, it may be having trouble with other ports as well.

The default web UI uses port 5000, but this is the same port that's used by Blue Iris's code project AI. I can try changing that port, but I know I had it working before on port 5001 using the following docker-compose.yml text (password / username redacted obviously)

version: '2.4'
services:
    wyze-bridge:
        container_name: wyze-bridge
        restart: unless-stopped
        image: mrlt8/wyze-bridge:latest
        ports:
            - 1935:1935 # RTMP
            - 8554:8554 # RTSP
            - 8888:8888 # HLS
            - 8889:8889 #WebRTC
            - 8189:8189/udp # WebRTC/ICE
            - 5001:5001 # WEB-UI
        environment:
            # [REQUIRED] Fill the following two lines:
            - WYZE_EMAIL=email
            - WYZE_PASSWORD=password
            # [OPTIONAL] IP Address of the host to enable WebRTC e.g.,:
            # - WB_IP=192.168.1.122

I recently had to purge my containers and images and I re-installed my wyze cam under a new name, and docker appears to show the bridge working, but it looks like it's trying to use the default port 5000 as well, and I don't understand why. Below is the output in my docker desktop log.

🚀 STARTING DOCKER-WYZE-BRIDGE v2.1.7

[WyzeBridge] 🔍 Could not find local cache for 'auth'
[WyzeBridge] ☁️ Fetching 'auth' from the Wyze API...
 * Serving Flask app 'frontend'
 * Debug mode: off
[WyzeBridge] WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://172.18.0.2:5000
[WyzeBridge] Press CTRL+C to quit
[WyzeBridge] 💾 Saving 'auth' to local cache...
[WyzeBridge] 🔍 Could not find local cache for 'user'
[WyzeBridge] ☁️ Fetching 'user' from the Wyze API...
[WyzeBridge] 💾 Saving 'user' to local cache...
[WyzeBridge] 🔍 Could not find local cache for 'cameras'
[WyzeBridge] ☁️ Fetching 'cameras' from the Wyze API...
[WyzeBridge] 💾 Saving 'cameras' to local cache...
[WyzeBridge] [+] Adding Float Cam [WYZE_CAKP2JFUS]
[WyzeBridge] starting MediaMTX v0.23.2
[WyzeBridge] 🎬 1 stream enabled

Edit: I have tried using the info from https://github.com/mrlt8/docker-wyze-bridge/wiki/WebUI#custom-ports but it hasn't made any difference, even after deleting my container and re-building.

**Comment by mrlt8** [comment, marked as answer]
You should be able to ignore that in the log messages as that's the internal port that it's listening on.

You'll need to update your yaml to point to the internal port:
```yaml
- 5001:5000 # WEB-UI
```
Note that the second port is **5000** as that's the port on the container which is listening on port 5000.

**Reply by shizenyasuya** [reply]
Thank you so much for this suggestion. I am re-installing docker desktop right now just to try it out. I thought there might have been something wrong with the port configuration, and I guess I must have over-written my previous working version with the generic version at some point. I'll update if this works or not.

**Reply by shizenyasuya** [reply]
You're amazing. Thank you. It's up and running now. It's silly I only have one wyze cam but thanks to the sunk-cost fallacy, I can't bring myself to get rid of it even though it takes this much extra effort and system resources to get running in blueiris.

**Comment by spamwax** [comment]
this change (`5001:5000`) was also necessary to run the sever on a macOS since port 5000 is already used by ControlCenter

```bash
▶ lsof -i:5000
COMMAND   PID  USER   FD   TYPE             DEVICE SIZE/OFF NODE NAME
ControlCe 434 dev     11u  IPv4   0xc147a515a95518      0t0  TCP *:commplex-main (LISTEN)
ControlCe 434 dev     12u  IPv6   0xe8e1cac731dcf025    0t0  TCP *:commplex-main (LISTEN)
```

**Comment by sircambridge** [comment]
I get that port 5000 is the internal port.

however my wyze setup has run into some issue where I have to run wyze-bridge in host networking mode as a workaround, not bridge, otherwise a lot of my cameras wont connect with the new firmware. https://github.com/mrlt8/docker-wyze-bridge/issues/1459

im running it all on a synology box, where the default port for the synology DSM is port 5000, so i have this unraveling problem. 

<img width="893" height="423" alt="image" src="https://github.com/user-attachments/assets/7b6c2f47-793a-40f1-902e-1b3e06dcd097" />
